spec/requests/locations_controller_spec.rb

```diff
@@ -215,7 +215,7 @@
       end
 
       it "has correct title" do
-        expected_title = CGI.escapeHTML("#{scheme.service_name} - Submit social housing lettings and sales data (CORE) - GOV.UK")
+        expected_title = CGI.unescapeHTML("#{scheme.service_name} - Submit social housing lettings and sales data (CORE) - GOV.UK")
         expect(page).to have_title(expected_title)
```

Comment:
In any future test written checking the title of the scheme might fail unless a developer follows the same approach taken here.